### PR TITLE
DOCS-5155 Service Scorecards beta docs

### PR DESCRIPTION
**Pull request by kayayarai**
### What does this PR do?
Documents Service Scorecards

### Motivation
DOCS-5155

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by kayayarai** [comment]
opened https://datadoghq.atlassian.net/browse/DOCS-5379 to track editorial review

File: content/en/tracing/service_catalog/scorecards.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  text: "Service Catalog"
+---
+
+{{< callout url="#" btn_hidden="true" >}}
+  Service Scorecards are in beta. Contact <a href="/help">Support</a> to have it enabled for your organization.
+{{< /callout >}}
+
+## Overview
+
+Service scorecards help you monitor, prioritize, plan, and communicate effectively to take informed actions that improve your service's health and performance. Each scorecard shows status for Production Readiness, Observability Best Practices, and Documentation & Ownership. All eligible services in Service Catalog are automatically evaluated against a set of pass-fail criteria.
```
Comment (alai97) [inline content/en/tracing/service_catalog/scorecards.md:18]:
```suggestion
Service scorecards help you monitor, prioritize, plan, and communicate effectively to take informed actions that improve your service's health and performance. Each scorecard shows the status for Production Readiness, Observability Best Practices, and Documentation & Ownership. All eligible services in the Service Catalog are automatically evaluated against a set of pass-fail criteria.
```

File: content/en/tracing/service_catalog/scorecards.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+## How services are evaluated
+
+On the [Service Catalog][1], click **Scorecards**. Click the section titles to see the rules that services are evaluated for.
```
Comment (alai97) [inline content/en/tracing/service_catalog/scorecards.md:22]:
```suggestion
Navigate to the [**Scorecards** page][8] in the [Service Catalog][6]. Click the section titles to see the rules that services are evaluated for.
```

File: content/en/tracing/service_catalog/scorecards.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+## Further reading
+
+{{< partial name="whats-next/whats-next.html" >}}
+
+[1]: https://app.datadoghq.com/services
+[2]: /monitors/service_level_objectives/
+[3]: https://app.datadoghq.com/monitors/recommended
+[4]: /tracing/services/deployment_tracking/
+[5]: /tracing/other_telemetry/connect_logs_and_traces/
```
Comment (alai97) [inline content/en/tracing/service_catalog/scorecards.md:88]:
```suggestion
[5]: /tracing/other_telemetry/connect_logs_and_traces/
[6]: /tracing/service_catalog/
[7]: /getting_started/tagging/unified_service_tagging/
[8]: https://app.datadoghq.com/services/scorecard
```

File: content/en/tracing/service_catalog/scorecards.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+: Defining a Team makes it easier for your on-call staff to know which team to escalate to in case a service they are not familiar with is the root cause of an issue. 
+
+Contacts defined
+: Defining contacts reduces the time it takes for your on-call staff to escalate to the owner of another service, helping you recover your services faster from outages and incidents.
+
+Code repos defined
+: Identifying code repositories enables your engineers to perform an initial investigation into an issue without having to contact the service's owning team. This improves collaboration and helps your engineers increase their overall understanding of integration points.
+
+Any docs defined
+: In the Service Catalog Other Links section, specify additional links to resources such as runbooks, dashboards, or other internal documentation. This helps with initial investigations and provides quick access to emergency remediation runbooks for outages and incidents.
```
Comment (alai97) [inline content/en/tracing/service_catalog/scorecards.md:64]: I'm not exactly sure where this section is located, but is fine for now~

File: content/en/tracing/service_catalog/scorecards.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+
+## How scores are calculated
+
+Each out-of-the-box scorecard (Production Readiness, Observability Best Practices, Ownership & Documentation) is made up of a default set of rules. These reflect simple pass-fail conditions. 
+
+Individual rules may have restrictions based on data availability. For example, deployment-related rules rely on the availability of version tags through APM Unified Service Tagging.
```
Comment (alai97) [inline content/en/tracing/service_catalog/scorecards.md:70]:
```suggestion
Individual rules may have restrictions based on data availability. For example, deployment-related rules rely on the availability of version tags through APM [Unified Service Tagging][7]. 
```

File: content/en/tracing/service_catalog/scorecards.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+Any docs defined
+: In the Service Catalog Other Links section, specify additional links to resources such as runbooks, dashboards, or other internal documentation. This helps with initial investigations and provides quick access to emergency remediation runbooks for outages and incidents.
+
+## How scores are calculated
+
+Each out-of-the-box scorecard (Production Readiness, Observability Best Practices, Ownership & Documentation) is made up of a default set of rules. These reflect simple pass-fail conditions. 
+
+Individual rules may have restrictions based on data availability. For example, deployment-related rules rely on the availability of version tags through APM Unified Service Tagging. 
+
+Each rule lists a score for the percentage of services that are passing. Each scorecard has an overall score percentage that totals how many services are passing, across all rules (not how many services are passing all rules).
```
Comment (alai97) [inline content/en/tracing/service_catalog/scorecards.md:72]: A formatting suggestion—take it or leave it!

```suggestion
Each rule lists a score for the percentage of services that are passing. Each scorecard has an overall score percentage that totals how many services are passing, across all rules—**not** how many services are passing all rules.
```

File: content/en/tracing/service_catalog/scorecards.md [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+
+## View service-level details and scores
+
+The scorecard summary is accessible in the Service Catalog **Explore** page under the **Score** column in the **Ownership** tab. Here, you can see how your specific service or subset of services is doing per each scorecard, and the rules within each.
```
Comment (alai97) [inline content/en/tracing/service_catalog/scorecards.md:76]:
```suggestion
The scorecard summary is accessible on the [**Explore** page][1] in the Service Catalog under the **Score** column in the **Ownership** tab. You can see how your specific service or subset of services is doing for each scorecard, and the rules within each. 
```

File: content/en/tracing/service_catalog/scorecards.md [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+
+Individual rules may have restrictions based on data availability. For example, deployment-related rules rely on the availability of version tags through APM Unified Service Tagging. 
+
+Each rule lists a score for the percentage of services that are passing. Each scorecard has an overall score percentage that totals how many services are passing, across all rules (not how many services are passing all rules).
+
+## View service-level details and scores
+
+The scorecard summary is accessible in the Service Catalog **Explore** page under the **Score** column in the **Ownership** tab. Here, you can see how your specific service or subset of services is doing per each scorecard, and the rules within each. 
+
+Click **View Details** from the scorecard hover, or open the Service details side panel to see the **Scorecards** tab, which lists all the scorecards, the rules, and that service's pass-fail score for each.
```
Comment (alai97) [inline content/en/tracing/service_catalog/scorecards.md:78]:
```suggestion
Click **View Details** from the scorecard, or open the service details side panel to see the **Scorecards** tab, which lists all the scorecards, the rules, and that service's pass-fail score for each rule.
```

**Review by alai97 — COMMENTED** [review]
Looks good, some copy suggestions!

File: content/en/tracing/service_catalog/scorecards.md (hunk 4 above)
Comment (kayayarai) [reply content/en/tracing/service_catalog/scorecards.md:64]: Yeah, when this is less beta-y, we'll link it to the instructions about Setting Other Links

File: content/en/tracing/service_catalog/scorecards.md [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+The production readiness score for all services (unless otherwise indicated) is based on these rules:
+
+Have any SLOs defined 
+: [Service Level Objectives (SLOs)][2] provide a framework for defining clear targets around application performance, which helps you provide a consistent customer experience, balance feature development with platform stability, and improve communication with internal and external users.
+
+Have any monitors defined
+: Monitors reduce downtime by helping your team quickly react to issues in your environment. Review [recommended monitors][3].
+
+Specified on-call
+: Improve the on-call experience for everyone by establishing clear ownership of your services. This will give your on-call engineers the correct point of contact during incidents, reducing the time it takes to resolve your incidents.
```
Comment (kayayarai) [inline content/en/tracing/service_catalog/scorecards.md:35]:
```suggestion
: Improve the on-call experience for everyone by establishing clear ownership of your services. This gives your on-call engineers the correct point of contact during incidents, reducing the time it takes to resolve your incidents. 
```

**Review by alai97 — APPROVED** [review]

**Pull request merged**